### ISSUE TITLE
Altitude reading 

### ISSUE THREAD

**Issue by paulkupfer** [dragino/LGT-92_-LoRa_GPS_Tracker#19]
The altitude value displays as close to 600m, which does not seem correct. The gateway elevation is close to 1500m and the device is not far away. 

Is the altitude value absolute meters from sea level?

**Comment by tormodvolden** [comment]
Yes, it is in absolute meters from sea level. It seems to work fine here at 400 m.
